sass/_component_autodoc.scss

```diff
@@ -3,3 +3,45 @@ dl.class {
     margin: 0 0 0.5rem 1.5rem;
   }
 }
+
+// colour highlighting for the classes
+dl.class {
+    span.sig-prename.descclassname {
+        color: $black;
+    }    
+}
```

Comment:
no need to duplicate the class here, instead incorporate these inside the `dl.class`

```
dl.class {
  .sig-object + dd {
    margin: 0 0 0.5rem 1.5rem;
  }
  .sig-prename {
      color: $black;
   }
}
```

Example - also avoid the `span` and added classes where possible - only add this specific selector if really needed.

Reply:
resolved